### PR TITLE
Slack webhook returns 400 missing_text_or_fallback_or_attachments #708

### LINKED ISSUE #708: Slack webhook returns 400 missing_text_or_fallback_or_attachments
Slack webhook return  **400** _missing_text_or_fallback_or_attachments_. The payload that hastic is sending

```
{
    "meta": {
        "type": "DETECT",
        "analyticUnitType": "ANOMALY",
        "analyticUnitName": "anomaly_booking",
        "analyticUnitId": "f6UQsg09lW0YbOji",
        "grafanaUrl": "http://grafana.trv/d/Tgliz?panelId=86&edit=true&fullscreen=true?orgId=1",
        "from": 1562346000000,
        "to": 1562346000000,
        "message": "2413.743119266055 out of bound"
    },
    "message": "[ANOMALY ALERTING] anomaly_booking\nURL: http://grafana.trv/d/TglViz?panelId=86&edit=true&fullscreen=true?orgId=1\n\nFrom: Fri Jul 05 2019 17:00:00 GMT+0000 (UTC)\nTo: Fri Jul 05 2019 17:00:00 GMT+0000 (UTC)\nID\nSummary: 2413.743119266055 out of bound\nInstance: perf-hastic0"
}
```

The configuration to get the payload is

```
HASTIC_WEBHOOK_URL = "http://httpdump.io/wzqre/"
HASTIC_WEBHOOK_TYPE = "application/json"
HASTIC_WEBHOOK_IMAGE_ENABLED = "true"
```

When replaced ```message```  by ```text``` in the payload, the slack webhook notification works as shown in the image below

{
    "meta": {
        "type": "DETECT",
        "analyticUnitType": "ANOMALY",
        "analyticUnitName": "anomaly_booking",
        "analyticUnitId": "f6UQsg09lW0YbOji",
        "grafanaUrl": "http://grafana.trv/d/Tgliz?panelId=86&edit=true&fullscreen=true?orgId=1",
        "from": 1562346000000,
        "to": 1562346000000,
        "message": "2413.743119266055 out of bound"
    },
    "text": "[ANOMALY ALERTING] anomaly_booking\nURL: http://grafana.trv/d/TglViz?panelId=86&edit=true&fullscreen=true?orgId=1\n\nFrom: Fri Jul 05 2019 17:00:00 GMT+0000 (UTC)\nTo: Fri Jul 05 2019 17:00:00 GMT+0000 (UTC)\nID\nSummary: 2413.743119266055 out of bound\nInstance: perf-hastic0"
}

hastic_webhook_notification

### PR DESCRIPTION
**Pull request by amper43**
fix #708 

related hastic-telegram-bot PR: https://github.com/hastic/hastic-telegram-bot/pull/2

### Changes
- just rename `message` to `text`

### CODE REVIEW

File: server/src/services/alert_service.ts [hunk 1]
```diff
@@ -187,15 +187,15 @@ export class AlertService {
     this._alerts[id].receive(segment);
   };
 
-  public sendMsg(message: string, type: WebhookType, optionalInfo = {}) {
+  public sendMsg(text: string, type: WebhookType, optionalInfo = {}) {
     const now = Date.now();
     const infoAlert: InfoMeta = {
```
Comment (jonyrock) [inline server/src/services/alert_service.ts]: maybe rename it to `metaInfo`

File: server/src/services/alert_service.ts [hunk 2]
```diff
@@ -187,15 +187,15 @@ export class AlertService {
     this._alerts[id].receive(segment);
   };
 
-  public sendMsg(message: string, type: WebhookType, optionalInfo = {}) {
+  public sendMsg(text: string, type: WebhookType, optionalInfo = {}) {
```
Comment (jonyrock) [inline server/src/services/alert_service.ts]: please rename it to `sendMessage`

File: server/src/services/alert_service.ts [hunk 3]
```diff
@@ -24,8 +24,8 @@ export class Alert {
 
   protected async makeNotification(segment: Segment): Promise<Notification> {
     const meta = this.makeMeta(segment);
-    const message = this.makeMessage(meta);
-    let result: Notification = { meta, message };
+    const text = this.makeMessage(meta);
+    let result: Notification = { meta, text };
     if(HASTIC_WEBHOOK_IMAGE_ENABLED) {
       try {
        const image = await this.loadImage();
```
Comment (jonyrock) [inline server/src/services/alert_service.ts]:
```suggestion
        const image = await this.loadImage();
```

File: server/src/services/analytics_service.ts [hunk 4]
```diff
@@ -182,14 +182,14 @@ export class AnalyticsService {
...
     }
     console.log(msg);
-    this._alertService.sendMsg(msg, WebhookType.RECOVERY);
+    this._alertService.sendMessage(msg, WebhookType.RECOVERY);
   }
 
   private async _onAnalyticsDown() {
     const msg = 'Analytics is down';
     console.log(msg);
     // TODO: enable analytics down webhooks when it stops bouncing
```
Comment (rozetko) [inline server/src/services/analytics_service.ts]:
```suggestion
```

**Pull request merged**